bench_autogptq/bench.sh

```diff
@@ -137,12 +136,8 @@ while [ "$#" -gt 0 ]; do
             fi
             shift 2
             ;;
-        -lf|--log_file)
-            LOG_FILENAME="$2"
-            shift 2
-            ;;
-        -md|--models_dir)
-            MODELS_DIR="$2"
+        -n|--model_name)
+            MODEL_NAME="$2"
```

Comment:
check supported models (e.g. only llama and mistral)

Reply:
This thing is supported inside bench.py base class :)

https://github.com/premAI-io/benchmarks/blob/4515ac19ea2d7d075b1a98df14a22b433cb3e25e/common/base.py#L34-L36